src/js/fee.js

```diff
@@ -125,10 +125,7 @@
 
       if (content !== 'raw') {
         returnContent = returnContent.replace(/<p>(?:<br ?\/?>|\u00a0|\uFEFF| )*<\/p>/g, '<p>&nbsp;</p>');
-        returnContent = returnContent.replace(/<p>\[/g, '[');
-        returnContent = returnContent.replace(/\]<\/p>/g, ']');
-        returnContent = returnContent.replace(/<br \/>/g, '');
-        returnContent = returnContent.replace(/<p>&nbsp;<\/p>/g, '<br />');
+        returnContent = window.switchEditors.pre_wpautop(returnContent);
```

Comment:
This is removes <p> and <br> tags which are not inside <pre> or <script> tags as similar to wpautop. But prewpautop focuses only on reduced set of elements when compared to wpautop. It prettifies content by removing extra spaces and adding newline characters.